Rubberduck.Parsing/VBA/RubberduckParserState.cs

```diff
@@ -697,7 +699,8 @@ public bool RemoveRenamedComponent(VBComponent component, string oldComponentNam
 
             if (success)
             {
-                OnStateChanged();
+                OnStateChanged(ParserState.ResolvedDeclarations);   // trigger test explorer and code explorer updates
+                OnStateChanged(ParserState.Ready);   // trigger find all references &c. updates
```

Comment:
Nobody, AFAIK.  These are all different places clearing the cache that do not involve reparses.  And I don't fire ResolvedDeclarations based on the modules, but based on the parser progress.